core/toolbox/category.ts

```diff
@@ -225,7 +225,6 @@ export class ToolboxCategory
    */
   protected createContainer_(): HTMLDivElement {
     const container = document.createElement('div');
-    container.tabIndex = -1;
```

Comment:
This is the cause of the bug @rachel-fenichel found.

The problem comes from the fact that there's no automatic focusing when clicking on categories now, whereas there was before. This results in there not being a focused node upon click, so toolbox defaults to its first category when it receives focus (since it has no child receive focus).

It's actually correct to put this line back as the fix. I'll add a line comment here & for separator to try and make it extra clear why this is needed.

Reply:
Fixed in latest.